src/components/content/catalog/services/update/UpdateResult.tsx

```diff
@@ -34,7 +34,7 @@ function UpdateResult({
                 message={
                     serviceRegistrationStatus === serviceTemplateRegistrationState.APPROVED ? (
                         <>
-                            Service <b>{ocl.name}</b> updated in catalog successfully.
+                            Service <b>{ocl.name}</b> updated was submitted for review.
```

Comment:
We can change as - 'Updated service template of service <b>{ocl.name}</b> is submitted for review.'